sample.env

```diff
@@ -1,4 +1,5 @@
 PORT=9090 # application's port
+METRICS_PORT=9091 # metrics's port
```

Comment:
Explain: Metrics (/metrics) will be served on a different port. This is done to support collection of aggregated metrics when transformer service is run in cluster mode on production.